dbt_loom/__init__.py

```diff
@@ -169,7 +196,24 @@ def read_config(self, path: Path) -> Optional[dbtLoomConfig]:
         if not path.exists():
             return None
 
-        return dbtLoomConfig(**yaml.load(open(path), yaml.SafeLoader))
+        with open(path) as file:
+            config_content = file.read()
+
+        config_content = self.replace_env_variables(config_content)
+
+        return dbtLoomConfig(**yaml.load(config_content, yaml.SafeLoader))
+
+    @staticmethod
+    def replace_env_variables(config_str: str) -> str:
+        """Replace environment variable placeholders in the configuration string."""
+        pattern = r'\$(\w+)|\$\{([^}]+)\}'
+        return re.sub(
+            pattern,
+            lambda match: os.environ.get(
+                match.group(1) if match.group(1) is not None else match.group(2), ''
+            ),
+            config_str,
+        )
```

Comment:
Adding support for environment variables in the config is quite clever and a great addition to the project! Can you please add some brief documentation to the README that shows this being used?